### DISCUSSION TITLE
(12) Stoxx [P.20160519]

### DISCUSSION THREAD

**Discussion by progettistamatto** [sapienzastudentsnetwork/basi-di-dati-2#29] [Progetti]
In questa discussion a tema il progetto [Stoxx](https://elearning.uniroma1.it/pluginfile.php/1350504/mod_folder/content/0/%285%29%20P%20-%20Progetti/%2812%29%20Stoxx%20-%20BD2.pdf), discuteremo di eventuali proposte di soluzione, dubbi o scelte progettuali in merito alla realizzazione del progetto in questione

**Comment by MattiaBaldinetti** [comment]
Ciao a tutti/e, vi lascio il link alla mia soluzione (corretta da Esposito). Se vi fa piacere lasciatemi una :star:: https://raw.githubusercontent.com/MattiaBaldinetti/Basi-di-Dati-2/03cd435d9b4a7395ab18665d9f18ef2b2eb3b6a3/PROGETTI/P%20-%20Stoxx.pdf
